commands/project/create.ts

```diff
@@ -28,36 +37,92 @@ exports.describe = uiBetaTag(i18n(`${i18nKey}.describe`), false);
 exports.handler = async options => {
   const { derivedAccountId } = options;
 
-  const hasCustomTemplateSource = Boolean(options.templateSource);
+  let latestRepoReleaseTag;
+  let templateSource = options.templateSource;
+  if (!templateSource) {
+    templateSource = HUBSPOT_PROJECT_COMPONENTS_GITHUB_PATH;
+    try {
+      const releaseData = await fetchReleaseData(
+        HUBSPOT_PROJECT_COMPONENTS_GITHUB_PATH
+      );
+      if (releaseData) {
+        latestRepoReleaseTag = releaseData.tag_name;
+      }
+    } catch (err) {
+      logger.error(i18n(`${i18nKey}.error.failedToFetchProjectList`));
+      process.exit(EXIT_CODES.ERROR);
+    }
+  }
 
-  let githubRef = '';
+  const projectTemplates = await getProjectTemplateListFromRepo(
+    templateSource,
+    latestRepoReleaseTag || DEFAULT_PROJECT_TEMPLATE_BRANCH
```

Comment:
Will this cause any issues if someone uses a custom template with a repo that uses something other than `main` (could see `master` still being common, for example)